app.ts

```diff
@@ -26,7 +26,7 @@ const main = async () => {
 
   const data = await initData();
 
-  client.v1.tweet(`I am deployed 👏👏👏`);
+  client.v1.tweet(`I am deployed 👏👏👏 - ${new Date().toISOString()}`);
```

Comment:
Twitter API doesn't allow duplicate tweets so sending the same deployed message could prevent you from starting the bot again for a while. While this isn't the best solution, this is mostly just to point out the issue.

Reply:
Thanks for pointing out the issue, ultimately I think we should implement a changelog tweet step in a CI. In the meantime we shall update this with a changelog in due time.